pkg/app/daemon/daemon.go

```diff
@@ -157,6 +159,26 @@ func NewExitImmediatelyInstruction() Instruction {
 	return &ExitImmediatelyInstruction{}
 }
 
+type FixTodosFileInstruction struct{}
```

Comment:
I'd rather call this what it is i.e. `RemoveUpdateRefsForCopiedBranch`. I don't consider the default behaviour broken and in need of fixing; rather I consider it a default that we don't like.

Reply:
Fine with me, changed in f9a285fb10e2 (along with other related symbols).